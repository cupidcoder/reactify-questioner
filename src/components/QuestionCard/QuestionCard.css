.QuestionCard {
  box-shadow: 6px 3px 11px 3px rgba(128,128,128,0.61);
  border-radius: 3px;
  padding: 1.5rem;
  margin-bottom: 1.5rem;
}

.QuestionContent {
  padding: 1rem;
}

.IconGroup {
  display: flex;
}

.IconGroup p {
  margin-top: 1rem;
  margin-right: 1rem;
  padding: 1rem 0;
}
