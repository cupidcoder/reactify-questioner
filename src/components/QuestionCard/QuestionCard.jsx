import React from 'react';
import style from './QuestionCard.css';

const QuestionCard = ({ question, commentCount }) => (
  <div className={style.QuestionCard}>
    <div className={style.QuestionContent}>
      <p>{question}</p>
      <div className={style.IconGroup}>
        <p>
          <i className='fas fa-arrow-up' />
          {' '}
          upvote
        </p>
        <p>
          <i className='fas fa-arrow-down' />
          {' '}
          downvote
        </p>
        <p>
          <i className='far fa-comments' />
          {' '}
          {commentCount}
        </p>
      </div>
    </div>
  </div>
);

export default QuestionCard;
