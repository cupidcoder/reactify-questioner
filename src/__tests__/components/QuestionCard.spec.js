import React from 'react';
import { shallow } from 'enzyme';
import QuestionCard from '../../components/QuestionCard/QuestionCard';

describe('QuestionCard', () => {
  it('Should correctly save snapshot of the QuestionCard', () => {
    const wrapper = shallow(<QuestionCard />);
    expect(wrapper).toMatchSnapshot();
  });
});
