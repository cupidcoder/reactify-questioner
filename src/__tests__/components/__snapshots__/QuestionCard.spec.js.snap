// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionCard Should correctly save snapshot of the QuestionCard 1`] = `ShallowWrapper {}`;
